R/changeLegendColour.R

```diff
@@ -18,77 +18,11 @@ changeLegendColour <- function(x,
     stop("x must be a ggplot object")
```

Comment:
Can't add the comment in the right place, but the documentation could stand to be a little more verbose here, in particular a few lines about what you can expect the legend to look like. Would also be handy to have an example or two included in the documentation (using the @example parameter), just duplicates of the ones from the readme would be fine, just easier for people to access!